R/utils-side.R

```diff
@@ -222,7 +222,10 @@ add_new_side <- function(object, side.name){
   stopifnot(length(side.name) == 1)
   if (side.name %in% side_names(object))
     stop(side.name, ' already exists, cannot add it.', call. = FALSE)
-  side.template <- list(list(lim = c(NA, NA), log=FALSE, label="", axes = TRUE, reverse = FALSE, usr.lim=c(FALSE, FALSE)))
+  side.template <- list(list(
```

Comment:
The side template now has a default axis call built in automatically that is overridden if a user calls `axis`